From 00f782ae25bbdc44ff72741971def65b4e9ec621 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 20 Mar 2019 12:54:55 +0800
Subject: [PATCH 609/706] enetc: EMU: replace TBIER/RBIER by SITXIER/SIRXIER
 for L0+ model

This change required by sim models m0170 and above, and by L0+ PXP models.
PDM ticket issued, expected to be fixed before TO.

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc_hw.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc_hw.h b/drivers/net/ethernet/freescale/enetc/enetc_hw.h
index 048601be660b..29066ace0077 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_hw.h
+++ b/drivers/net/ethernet/freescale/enetc/enetc_hw.h
@@ -84,10 +84,10 @@ static inline u32 enetc_vsi_set_msize(u32 size)
 
 #define ENETC_PSIIER	0xa00
 #define ENETC_PSIIER_MR_MASK	GENMASK(2, 1)
-#define ENETC_PSIIDR	0xa10
+#define ENETC_PSIIDR	0xa08
 #define ENETC_SITXIDR	0xa18
-#define ENETC_SIRXIDR	0xa1c
-#define ENETC_SIMSIVR	0xa20
+#define ENETC_SIRXIDR	0xa28
+#define ENETC_SIMSIVR	0xa30
 
 #define ENETC_SIMSITRV(n) (0xB00 + (n) * 0x4)
 #define ENETC_SIMSIRRV(n) (0xB80 + (n) * 0x4)
-- 
2.17.1

